tests/pool_test.go

```diff
@@ -92,16 +92,20 @@ var _ = Describe("[sig-compute]VirtualMachinePool", decorators.SigCompute, func(
 		pool, err := virtClient.VirtualMachinePool(util.NamespaceTestDefault).Patch(context.Background(), name, types.JSONPatchType, []byte(fmt.Sprintf("[{ \"op\": \"replace\", \"path\": \"/spec/replicas\", \"value\": %v }]", scale)), metav1.PatchOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
+		running := *pool.Spec.VirtualMachineTemplate.Spec.Running
```

Comment:
What if the VM has the `RunStrategy` instead of the `Running` field? Thanks

Reply:
@fossedihelm Hi! in the scope of these tests only `pool.Spec.VirtualMachineTemplate.Spec.Running` is being used. Would like me to include the run strategy check nevertheless?

Reply:
I think it is enough to leave things like this. Thanks